docs/examples/sock-shop.md

```diff
@@ -295,15 +303,13 @@ Path: ./deploy/kubernetes/manifests
 Branch: master
 Type: kustomize
 
-◎ Checking cluster status
-✔ Wego installed
 ✚ Generating Source manifest
 ✚ Generating GitOps automation manifests
 ✚ Generating Application spec manifest
 ► Writing manifests to disk
 ► Applying manifests to the cluster
-► Committing and pushing wego updates for application
-► Pushing app changes to repository
+► Committing and pushing gitops updates for application
+✔ App is up to date
```

Comment:
not sure if we need to reset the config repo to avoid `App is up to date`  and have `Pushing app changes to repository`